### PR TITLE
[testing-devel] overrides: Fast-track podman 3.1.0-2

### PR DESCRIPTION
**Pull request by travier**
For https://github.com/coreos/fedora-coreos-tracker/issues/771

### CODE REVIEW

**Review by jlebon — DISMISSED** [review]
Thanks!

File: manifest-lock.overrides.yaml [hunk 1]
```diff
@@ -36,3 +36,10 @@ packages:
     evra: 0.21.2-1.fc33.noarch
   console-login-helper-messages-profile:
     evra: 0.21.2-1.fc33.noarch
+  # Fast-track new podman release to fix podman cp:
+  # https://github.com/coreos/fedora-coreos-tracker/issues/771
+  # https://bodhi.fedoraproject.org/updates/FEDORA-2021-d8a3be0829
+  podman:
+    evra: podman-3.1.0-1.fc33
+  podman-plugins:
+    evra: podman-plugins-3.1.0-1.fc33
```
Comment (sohankunkerkar) [inline manifest-lock.overrides.yaml]: Shouldn't this be `evr` instead?

Comment (jlebon) [reply manifest-lock.overrides.yaml]: Yes, good catch! We all need to get used to this new model (and then to https://github.com/coreos/fedora-coreos-config/pull/906).

**Comment by travier** [comment]
Oops, updated!

**Comment by dustymabe** [comment]
`podman-3.1.0-2.fc33 ` obsoleted `3.1.0-1`: https://bodhi.fedoraproject.org/updates/FEDORA-2021-e70b450680

File: manifest-lock.overrides.yaml [hunk 2]
```diff
@@ -36,3 +36,10 @@ packages:
     evra: 0.21.2-1.fc33.noarch
   console-login-helper-messages-profile:
     evra: 0.21.2-1.fc33.noarch
+  # Fast-track new podman release to fix podman cp:
+  # https://github.com/coreos/fedora-coreos-tracker/issues/771
+  # https://bodhi.fedoraproject.org/updates/FEDORA-2021-d8a3be0829
```
Comment (dustymabe) [inline manifest-lock.overrides.yaml]:
```suggestion
  # https://bodhi.fedoraproject.org/updates/FEDORA-2021-e70b450680
```

File: manifest-lock.overrides.yaml [hunk 3]
```diff
@@ -36,3 +36,10 @@ packages:
...
+  podman:
+    evr: 3.1.0-2.fc33
```
Comment (dustymabe) [inline manifest-lock.overrides.yaml]:
```suggestion
    evr: 2:3.1.0-2.fc33
```

File: manifest-lock.overrides.yaml [hunk 4]
```diff
@@ -36,3 +36,10 @@ packages:
...
+  podman-plugins:
+    evr: 3.1.0-2.fc33
```
Comment (dustymabe) [inline manifest-lock.overrides.yaml]:
```suggestion
    evr: 2:3.1.0-2.fc33
```

**Review by dustymabe — APPROVED** [review]
LGTM

**Pull request merged**